benchmarks/Microsoft.AspNetCore.SignalR.Microbenchmarks/Microsoft.AspNetCore.SignalR.Microbenchmarks.csproj

```diff
@@ -12,8 +12,11 @@
     <ProjectReference Include="..\..\src\Microsoft.AspNetCore.SignalR.Common\Microsoft.AspNetCore.SignalR.Common.csproj" />
     <ProjectReference Include="..\..\src\Microsoft.AspNetCore.SignalR.Client.Core\Microsoft.AspNetCore.SignalR.Client.Core.csproj" />
     <ProjectReference Include="..\..\src\Microsoft.AspNetCore.SignalR.Protocols.MessagePack\Microsoft.AspNetCore.SignalR.Protocols.MessagePack.csproj" />
-    <ProjectReference Include="..\..\src\Microsoft.AspNetCore.SignalR.Redis\Microsoft.AspNetCore.SignalR.Redis.csproj" />
     <ProjectReference Include="..\..\test\Microsoft.AspNetCore.SignalR.Tests.Utils\Microsoft.AspNetCore.SignalR.Tests.Utils.csproj" />
+    <ProjectReference Include="..\..\src\Microsoft.AspNetCore.SignalR.Redis\Microsoft.AspNetCore.SignalR.Redis.csproj">
+      <!-- Alias is required because internal MemoryBufferWriter is in multiple projects that InternalsVisibleTo this project -->
+      <Aliases>redis</Aliases>
```

Comment:
Ughh wtf.

Reply:
Maybe we should just pubternal instead?

Reply:
So leave Microsoft.AspNetCore.SignalR.Redis.Internal as public?

btw with these shared internal files, and InternalVisibleTo, having some aliases will eventually have to happen unless something changes. MemoryBufferWriter is used everywhere